src/components/Input/Checkbox/story.tsx

```diff
@@ -0,0 +1,144 @@
+      onChange={(e) => {
+        setValue(e.target.value);
+        setChecked(e.target.checked);
+        onChange();
+      }}
+      value={value !== undefined ? value : undefined}
+    />
+  );
+};
+
+storiesOf('Planets/Input/Checkbox', module)
+  .addParameters({
+    readme: {
+      sidebar: Readme,
+    },
+  })
+  .add(
+    'Overview',
+    () => (
+      <Wrap>
+        {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
+        <label className="text-xl inline-block mb-2">
+          {`<Checkbox />`}
+          <p className="mt-6">
+            <small>
+              <em>
+                Checkbox is a bare input. You&apos;ll have to provide your own
+                label. &ldquo;Control&rdquo; components with integrated labels
+                and validation messages are on the roadmap.
+              </em>
+            </small>
+          </p>
+          <div>
+            <Checkbox
+              checked={boolean('checked', false)}
+              className={text('className (HTML)', 'w-full')}
+              fauxDisabled={boolean('fauxDisabled', false)}
+              id="SamplCheckbox"
+              onChange={action('onChange (HTML)')}
+            />
+          </div>
+        </label>
+      </Wrap>
+    ),
+    {
+      info: {
+        inline: true,
+        source: true,
+      },
+    }
+  )
+  .add(
+    'Examples',
+    () => (
+      <Wrap>
+        <h2>Checkbox</h2>
+        <p className="mt-6">
+          <small>
+            <em>
+              Checkbox is a bare input - the labels are provided for
+              documentation purposes only. &ldquo;Control&rdquo; components with
+              integrated labels and validation messages are on the roadmap.
+            </em>
+          </small>
+        </p>
+        <p className="mt-6">
+          <small>
+            This component is identical to {'`<Input type="checkbox" />`'}.
+          </small>
+        </p>
+        <p className="mt-6">
+          <small>
+            These examples use an `InteractiveInput` wrapper with an
+            `InputRenderer` prop. In actual implementations, just use a
+            `Checkbox` component, `onChange`, and `checked` props instead.
+          </small>
+        </p>
+        <div>
+          {[
+            [],
+            ['checked'],
+            ['disabled'],
+            ['fauxDisabled'],
+            ['checked', 'disabled'],
+          ].map((attributes) => (
+            <div className="my-6" key={['atts', ...attributes].join('-')}>
+              <label htmlFor={['atts', ...attributes].join('-')}>
+                {attributes.join(' and ') || 'default'}{' '}
+                <InteractiveInput
+                  InputRenderer={Checkbox}
+                  id={['atts', ...attributes].join('-')}
+                  onChange={action(
+                    `onChange ${attributes.join(' and ') || 'default'}`
+                  )}
+                  {...attributes.reduce(
```

Comment:
Not super legible. Maybe I should write a helper for this? Or just not do immediate function return, so I can give this generated block of truthy props a proper name.

The upshot is: For an array like `['checked', 'disabled']`, it should produce a props value like `{ checked: true, disabled: true }`, so I can produce stories for a whole buncha variants easily.